index.html

```diff
@@ -252,14 +256,27 @@
 
     // Filter Occupations to Major Group Types
     if (options.filtered) {
-        data = data.filter(function (d) { return d.Group === options.filtered; })
+        data = data.filter(function (d) { return d.Group === options.filtered; });
     } 
 
     // Select only the occupations from the current page (slice)
     if (options.page) {
-      data = data.slice(options.page*20+1,options.page*20+21 )
+      data = data.slice(options.page*20+1,options.page*20+21 );
+      	if (data[options.page*20+22] ===null) {
```

Comment:
An item that doesn't exist in an array is `undefined`, so we can test it like this: `typeof data[n] === 'undefined'`

But we can also take advantage of the fact that `undefined` is falsey, and simply write `if (!data[n])`.